event-grid/subscribe-to-custom-topic-preview/subscribe-to-custom-topic-preview.ps1

```diff
@@ -0,0 +1,21 @@
+# You must have the latest version of the Event Grid PowerShell module.
+# To install:
+# Install-Module -Name AzureRM.EventGrid -AllowPrerelease -Force -Repository PSGallery
+
+# Provide the name of the topic you are subscribing to
+$myTopic = "demoContosoTopic"
+
+# Provide an endpoint for handling the events.
+$myEndpoint = "<endpoint URL>"
+
+# Provide the name of the resource group containing the custom topic.
+$myResourceGroup = "demoResourceGroup"
```

Comment:
Examples should be consistent in whether they predefine a resource group or rely on a user change.